coretex/_task/local/arg_parser.py

```diff
@@ -37,6 +37,9 @@ class LocalArgumentParser(Tap):
     name: Optional[str]
     description: Optional[str]
 
+    saveSnapshot: bool
+    entryPoint: Optional[str]
```

Comment:
When running local experiments it is currently not possible to specify a dynamic entry point, as it always goes through the "_coretex.py" entry point. We can safely remove this, and properly implement a dynamic entry point for local experiments if it is needed.